mmr/runner.rs

```diff
@@ -49,13 +62,21 @@ impl Runner {
     /// Start the runner
     pub fn start(&mut self) -> Result<(), Error> {
         match self.mmr_count() {
-            Ok(mut base) => {
-                base = Runner::mmr_size_to_last_leaf(base);
+            Ok(mmr_count) => {
+                let mut next = {
+                    let last_leaf = Runner::mmr_size_to_last_leaf(mmr_count);
+                    if last_leaf == 0 {
+                        0
+                    } else {
+                        last_leaf + 1
+                    }
+                };
```

Comment:
Here is a bug that exists before, the MMR service pushed the block which is after the first **service break** twice times, and the continuance of the block mmr breaks after that.